pulpcore/pulpcore/app/models/__init__.py

```diff
@@ -2,8 +2,14 @@
 
 from .auth import User  # noqa
 from .base import Model, MasterModel  # noqa
-from .generic import (GenericRelationModel, GenericKeyValueManager, GenericKeyValueRelation,  # noqa
-                      GenericKeyValueModel, Config, Notes)  # noqa
+from .generic import (  # noqa
+    GenericRelationModel,
```

Comment:
Since you are reformatting this, mind doing alphabetical order?

Reply:
I think it's this way due to circular imports

Reply:
Cool. If you are sure that this would cause circular imports, please add a comment. Don't worry about it if you aren't, didn't mean to scope creep you.